aviator.js

```diff
@@ -1028,6 +1044,23 @@ Route.prototype = {
   **/
   isNamedParam: function (fragment) {
     return fragment.indexOf('/:') === 0;
+  },
+
+  /**
+  Assign the given function to the next sequential key in the this.anonymous
+  object.
+
+  @method generateNotFoundAction
+  @param {Function} func the anonymous function to transform into an action.
+  @return {Object} With a method key that maps to a generated String, and
+  **/
+  anonymousAction: function (func) {
+    var method = 'FUNCTION-' + Object.keys(this.anonymous).length;
```

Comment:
Wouldn't the timestamp be more likely to collide than this? Being that JS is single-threaded, I don't think this can collide. :+1: on resetting `this.anonymous`.